synapse/models/inet.py

```diff
@@ -1535,16 +1538,19 @@ def getModelDefs(self):
                             'doc': 'The HTTP query string which optionally follows the path.'}),
 
                         ('headers', ('array', {'type': 'inet:http:request:header'}), {
-                            'doc': 'An array of HTTP headers from the request.'}),
-
+                            'doc': 'An array of HTTP headers from the request.',
+                            'disp': {'hint': 'text'},
+                        }),
                         ('body', ('file:bytes', {}), {
                             'doc': 'The body of the HTTP request.'}),
 
                         ('response:time', ('time', {}), {}),
                         ('response:code', ('int', {}), {}),
                         ('response:reason', ('str', {}), {}),
                         ('response:headers', ('array', {'type': 'inet:http:response:header'}), {
-                            'doc': 'An array of HTTP headers from the response.'}),
+                            'doc': 'An array of HTTP headers from the response.',
+                            'disp': {'hint': 'text'},
+                        }),
```

Comment:
we should probably make a call ( and a ticket to normalize ) the declaration syntax having the closing }) on the last info line or on it's own line.  :D

Reply:
yep, agreed.